### DISCUSSION TITLE
Compatible with 1.20 please

### DISCUSSION THREAD

**Discussion by iSeitan** [Kas-tle/java2bedrock.sh#8378] [Feature Requests]
the converter is not compatible with 1.20 yet

**Comment by munchkinth** [comment]
Why is it not supported in 1.20?

**Comment by HaiXi2912** [comment]
Perhaps Kastle is busy, may he be busy with something else, or he may be adding new conversion types, such as armors, bows, shields, and customblock, thus delaying compatibility with 1.20 We need to trust Kastle
